src/Components/WebAssembly/WebAssembly/test/Hosting/EntrypointInvokerTest.cs

```diff
@@ -1,153 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Do we want to keep/tweak these tests to make sure things still work the same? Or are these behaviors/tests covered already elsewhere?

Reply:
We have E2E tests validating this.

Reply:
I don't believe these tests should be necessary any longer as they were meant to test out the functionality of the `EntrypointInvoker` specifically, which has now been removed. The core logic of finding the assembly and entrypoint is now handled on the runtime layer. Note; we do still have the E2E tests for the functionality: https://github.com/dotnet/aspnetcore/blob/main/src/Components/test/E2ETest/Tests/StartupErrorNotificationTest.cs